### DISCUSSION TITLE
MacOS

### DISCUSSION THREAD

**Discussion by ekbiker** [bhowiebkr/laser-level-webcam#21] [General]
I managed to get this working on Mac OS. pip didn't work so I had to install from source. Several things to note: 
1. the tool tip background on mac is dark, so the text doesn't show up very well. I changed the text color to 'white' in main.py and that took care of that.
2. pip threw error on incompatible python version, even through my 'python --version' and 'python3 --version' both returned 3.10.10.
3. FFMpeg requirement on mac is problematic, can the python-ffmpeg be used instead?
4. Readme is outdated

**Comment by ekbiker** [comment]
Will you accept pull requests if I updated the ffmpeg for macos.

**Comment by bhowiebkr** [comment]
For sure. Exact Python version is probably less important now. It was before when I was compiling to an exe and the dependencies needed exact versions of python and qt. 

FFmpeg is only used to start up a device window on the windows platform and might not work the same in Mac. 

Dark theme dependency has been a little buggy with tooltips.
